### ISSUE TITLE
design an affordance that would reset the zoom level to default 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/lenape-timetree#136]

**Comment by gissoo** [comment]
@rlskoeser @jhimpele Rebecca and I discussed that on mobile we would have a [button to let users reset the zoom level,](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1304%3A1246&t=16Yj0q6JutyqKogd-1) 

how it works: [if you're zoomed in on the tree you would press a button to view the entire tree again as it was displayed by default. ](https://www.figma.com/file/sGOn9MZy6ML0yNxuiQJ8u5/Lenape-Time-Tree?node-id=1304%3A1623&t=16Yj0q6JutyqKogd-1)(the button snaps the tree back to the container). Because of lack of space I'd strongly suggest we don't include text in the button such as "reset". 

I'm proposing this icon which will be intuitive in context. @rlskoeser can we add a bit more invisible padding to it so it will be accessible? I think there is a way. It's now 40px, needs 8 more to be accessible. I don't want to make it huge. 

Note: the icon is taken from phosphor icons, @rlskoeser

**Comment by jhimpele** [comment]
@gissoo @rlskoeser that button looks to like it reverses out of a full screen option, which we don't have of course. And to back up a step, what is the zoom in interaction?

**Comment by gissoo** [comment]
@jhimpele That's what it kind of does since it's snaps the tree back to its default position! It's more similar to zooming out. The zooming in interaction is by using two fingers and moving them away from each other, like below, you could zoom out by doing the opposite.
<img width="481" alt="Screenshot 2023-04-05 at 2 52 14 PM" src="https://user-images.githubusercontent.com/44615389/230177277-a1deca46-6ed5-4cbc-a9c1-4e90759d8cb6.png">

**Comment by jhimpele** [comment]
I get it! So it is for mobile. Thanks!


Jeffrey Himpele.
Director, VizE Lab for Ethnographic Data Visualization
Lecturer in Anthropology
vizelab.princeton.edu<http://vizelab.princeton.edu/>


From: gissoo ***@***.***>
Date: Wednesday, April 5, 2023 at 2:52 PM
To: Princeton-CDH/lenape-timetree ***@***.***>
Cc: Jeffrey D. Himpele ***@***.***>, Mention ***@***.***>
Subject: Re: [Princeton-CDH/lenape-timetree] design an affordance that would reset the zoom level to default (Issue #136)

@jhimpele<https://github.com/jhimpele> That's what it kind of does since it's snaps the tree back to its default position! It's more similar to zooming out. The zooming in interaction is by using two fingers and moving them away from each other, like below, you could zoom out by doing the opposite.
[Screenshot 2023-04-05 at 2 52 14 PM]<https://user-images.githubusercontent.com/44615389/230177277-a1deca46-6ed5-4cbc-a9c1-4e90759d8cb6.png>

**Comment by gissoo** [comment]
@jhimpele yes, we are enabling zoom only on mobile since all the leaves are visible on desktop at once.

**Comment by rlskoeser** [comment]
@gissoo this looks great! I agree that it should be pretty obvious in context, since it will only show up once you are zoomed in. 

Just to confirm, is this the "arrows-in" phosphor icon? And what style/weight of Phosphor icon? You didn't specify before I think I just used the regular.

Is 40px not enough for a tap target now, it needs to be 48x48? I should be able to add padding to make that work.

**Comment by gissoo** [comment]
@rlskoeser I'm sorry! Regular should be fine for the previous icons, this one is "arrows-in-fill". 
You can inspect its structure in the container, also – the square it's sitting in is 40x40 and there is a 9px padding on all sides for the icon within the square.

Thank you! Yes, according to my recent findings 40px is not enough.
